core/src/mindustry/entities/comp/BuildingComp.java

```diff
@@ -384,6 +384,10 @@ public byte version(){
     //endregion
     //region handler methods
 
+    public boolean unloadable(){
+        return block.unloadable;
```

Comment:
redundant check inside `Unloader.java` though, should this just be hardcoded to `true` or simply remove the block level check in unloader and route it all through this function?